bedrock/pocket/templates/pocket/get-inspired.html

```diff
@@ -25,11 +25,11 @@
   media_class='mzp-l-split-h-center',
   image_url='img/pocket/save-inspirations.png',
   )%}
-  <h1 class="section-heading">Save what inspires you</h1>
```

Comment:
Issue (blocking): page title above is untranslated

`{% block page_title %}Get Inspired{% endblock %}`